### DISCUSSION TITLE
What's the recommended way to give developers on my team access to upload to Supabase storage?

### DISCUSSION THREAD

**Discussion by ryansburgoyne** [supabase/supabase#29939] [Questions] [answered]
The developers on my team can't upload to storage. 

<img width="837" alt="Screenshot 2024-10-16 at 11 01 35 AM" src="https://github.com/user-attachments/assets/e6bd9f07-e2e5-4fd3-9d8d-fecbbc7c26bb">

How can I give them access? I think I need to use RLS policies, but how can I limit the permissions to logged in users on my team, not users in my database?

**Comment by GaryAustin1** [comment, marked as answer]
I think they just broke and are fixing this:
https://github.com/supabase/supabase/pull/29937

RLS would have nothing to do with your dashboard users.
